www/survey.html

```diff
@@ -5,8 +5,9 @@
 <meta charset="UTF-8">
 <title>Caterpillars Count!</title>
 <script src="./js/jquery-1.11.3.min.js"></script>
-<script src="cordova.js"></script>
-<script src="./js/input-page.js"></script>
+<!--<script src="cordova.js"></script>-->
+<!--<script src="./js/input-page.js"></script>-->
```

Comment:
I am not using input-page.js right now since it will check for other criteria in order to submit. I am only creating a button to simulate the offline mode. The cordova.js line can be recovered